components/NavBar.tsx

```diff
@@ -36,21 +36,20 @@ const NavBar: React.ComponentType = () => {
   const router = useRouter();
   const theme: typeof defaultTheme = useTheme();
 
-  const { NEXT_PUBLIC_LAB_NAME, NEXT_PUBLIC_LOGO_FILENAME, NEXT_PUBLIC_BASE_PATH } = getConfig();
+  const { NEXT_PUBLIC_LAB_NAME } = getConfig();
 
   const activeLinkStyle = `
     background-color: ${theme.colors.grey_2};
     color: ${theme.colors.accent2_dark};
   `;
 
-  const labIcon = NEXT_PUBLIC_LOGO_FILENAME ? (
-    <img
-      src={`${NEXT_PUBLIC_BASE_PATH}/static/dms_user_assets/${NEXT_PUBLIC_LOGO_FILENAME}`}
+  const labIcon = (
+    <Image
+      src={`/static/dms_logo.png`}
```

Comment:
This works fine for Eucancan but it would be good to know why the built in DMS system failed.